doc/user_guide/encodings/channels.rst

```diff
@@ -0,0 +1,187 @@
+=======  ================  ========================  =========================================
+
+.. _hyperlink-channel:
+
+Hyperlink
+~~~~~~~~~
+
+=======  ================  ========================  =========================================
+Channel  Altair Class      Description               Example
+=======  ================  ========================  =========================================
+href     :class:`Href`     Hyperlink for  points     :ref:`gallery_scatter_href`
+=======  ================  ========================  =========================================
+
+Detail
+~~~~~~
+
+Grouping data is an important operation in data visualization. For line and area marks,
+mapping an unaggregated data field to any
+non-position channel will group the lines and stacked areas by that field.
+For aggregated plots, all unaggregated fields encoded are used as grouping fields
+in the aggregation (similar to fields in ``GROUP BY`` in SQL).
+
+The ``detail`` channel specifies an additional grouping field (or fields) for grouping
+data without mapping the field(s) to any visual properties.
+
+=======  ================  ===============================  =========================================
+Channel  Altair Class      Description                      Example
+=======  ================  ===============================  =========================================
+detail   :class:`Detail`   Additional property to group by  :ref:`gallery_ranged_dot_plot`
+=======  ================  ===============================  =========================================
+
+For example here is a line chart showing stock prices of 5 tech companies over time.
+We map the ``symbol`` variable to ``detail`` to use them to group lines.
+
+.. altair-plot::
+    
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.stocks()
+    alt.Chart(source).mark_line().encode(
+        x="date:T",
+        y="price:Q",
+        detail="symbol:N"
+    )
+
+
+Order
+~~~~~
+
+The `order` option and :class:`Order` channel can sort how marks are drawn on the chart.
+
+For stacked marks, this controls the order of components of the stack. Here, the elements of each bar are sorted alphabetically by the name of the nominal data in the color channel.
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+
+    barley = data.barley()
+
+    alt.Chart(barley).mark_bar().encode(
+        x='variety:N',
+        y='sum(yield):Q',
+        color='site:N',
+        order=alt.Order("site", sort="ascending")
+    )
+
+The order can be reversed by changing the sort option to `descending`.
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+
+    barley = data.barley()
+
+    alt.Chart(barley).mark_bar().encode(
+        x='variety:N',
+        y='sum(yield):Q',
+        color='site:N',
+        order=alt.Order("site", sort="descending")
+    )
```

Comment:
```suggestion
    )

If we want to sort stacked segments in a custom order, we can `follow the approach in this issue comment <https://github.com/altair-viz/altair/issues/245#issuecomment-748443434>`_, although there might be edge cases where this is not fully supported. This also makes the order of the segments align with the order colors shows up in a legend that uses custom sorting for the color domain.

```

What do you think of including this?

Reply:
Cool, didn't know about this created field! I'll add your suggestion